wiki/CLI-language.md

```diff
@@ -407,6 +421,36 @@ The description will initially be defined the same as the slug, but can be modif
 
 After the tag is created, it can be seen in /Logical/Tags. The command `get /Logical/Tags/[slug]` can be used to get the tag information. In doing so, the tag image will be the route in which the image can be obtained via an http request.
 
+## Create a Layer
+
+Layers are identified by a slug. In addition, they have an applicability and the filters they apply. To create a layer, use:
+
+```
++layer:[slug]@[applicability]
+```
+
+The applicability is the path in which the layer should be added when doing ls.
+
+`*` and `**` can be used in the definition of the applicability so that the layer will be added in multiple paths. `*` represents any child, e.g. an applicability of /Physical/BASIC/* implies that the layer will be added to all the children of /Physical/BASIC.  `**` represents any child recursively, e.g. an applicability of /Physical/BASIC/** implies that the layer will be added to all children of /Physical/BASIC, to the children of the children and so on.
+
+Filters are automatically created as empty. To add filters, edit the layer using the object modification syntax (see [Modify object's attribute](#modify-objects-attribute)). Example:
+
+```
+[layer_slug]:category=device
```

Comment:
maybe explicit that it is path+slug or /Logical/Layers/layer_slug, if not in that path

Reply:
fixed in bc62dd10